### DISCUSSION TITLE
breakpoint doesn't work with custom style

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [tailwindlabs/tailwindcss#8286] [Help]
I added a `btn` class to style the button, and I want it to take effect at `md` breakpoint, but it didn't work, is there any other config I need to change ?
demo :  https://play.tailwindcss.com/SJbgXb9Owd?file=css
I added my `btn` class in the CSS tab

**Comment by Lelectrolux** [comment]
Read about the `@layer` directive : https://tailwindcss.com/docs/functions-and-directives#layer

Raw css is left untouched by default, unless you include it in a layer (`base`. `components`, or `utilities`).

Here is a working demo : https://play.tailwindcss.com/cVcRQikt5w?file=css

A `.btn` class would go in the `components` layer.

**Reply by Chandler-Zhu** [reply]
One follow up question, when I use the `screen()` function inside `@layer` it gives me some error, also the text color doesn't change when I reach the `md` breakpoint
![image](https://user-images.githubusercontent.com/61914365/167198638-1ab7f372-80ba-4fae-946f-889bd8485bd3.png)
